css/settings.css

```diff
@@ -1,58 +1,182 @@
-tr.spaceUnder > td
-{
-  padding-bottom: 1em;
-}
-label{
-  margin-left: 18px;
-}
-input[type='checkbox']{
-  margin-left: -18px;
-}
-.container
-{
-  margin: 2px;
-  width: auto;
-  height: 45px;
-  line-height: 45px;
-  text-align: center;
-}
-.resize_fit_center
-{
-  max-width:100%;
-  max-height:100%;
-  vertical-align: middle;
-}
-.settings{
-  line-height: 20px;
-  text-align: left;
-  max-width: 450px;
-  margin: auto;
-}
-.btn-docs{
-  background-color: gray;
-  color:white;
-  border-radius: 50%;
-  border:none;
-  transform: scale(.7);
+#settingPage {
+    height: 458px;
+    max-height: 458px;
+    overflow: hidden;
 }
-.tooltip-inner p{
-  margin: 10px 0 10px 0;
+
+#general {
+    font-size: 12px;
+    font-weight: initial;
 }
-#show_context{
-  padding-top: 5px;
-  font-size: 17px;
+
+#context {
+    font-size: 12px;
+    font-weight: initial;
 }
 
-#wrap {
-  width: 100%;
-  height:auto;
-  border-style:groove;
-  border-color:rgba(121, 197, 165, 0.24);
-  border-radius: 20px;
-  padding-bottom: 20px;
-  padding-left: 5px;
-  margin: 2px;
+button:focus {
+    outline: none !important;
 }
-label {
-  font-size: 14px;
+
+/* navgation bar */
+ul.nav {
+    list-style-type: none;
+    margin: 0px;
+    padding: 0px;
+    overflow: hidden;
 }
+
+li.section {
+    float: left;
+    margin-left: 2px;
+    margin-top: 5px;
+    border-width: 0 0 1px 0;
+    border-radius: 4px;
+}
+
+li.section span.section {
+    display: block;
+    color: white;
+}
+
+span.section {
+    text-align: center;
+    padding-left: 18px;
```

Comment:
Instead of addressing padding of each, please use padding shorthand property with four values.
See this https://www.w3schools.com/css/css_padding.asp .
You could see this at a lot of places, please make the changes.
Thank you !

Reply:
Will fix that.